phys/module_mp_ntu.F

```diff
@@ -6203,7 +6203,7 @@ SUBROUTINE LARGE_DT(DT,TK1D,QV1D,P1D,RHO,QC1D,QR1D,QI1D,QS1D,    &
             ENDIF
             QCLsh = QCLS1*ESH*VTQSH*NH1D*(SASR2*GSM3+SASR1*2.*GH2*GSM2+&
                     GH3*GSM1)
-            NCLsh = NCLS1*ESH*VTNSH*NH1D*(SASR2*GS3*+SASR1*2.*GH2*GS2+ &
+            NCLsh = NCLS1*ESH*VTNSH*NH1D*(SASR2*GS3 +SASR1*2.*GH2*GS2+ &
```

Comment:
@weiwangncar @dudhia 
This one is the biggy

Reply:
@davegill This looks good to me. I'll ask the author to approve it too.

Reply:
Thanks Wei.
Could we also ask the author to comment on the likely impact of multiplying, instead of adding those terms? That might be significant.

Reply:
@davegill I sent him this page, and hopefully he will see your comment.

Reply:
Hi, thanks. It is a typo. The quick evaluation of its impact is quite minor from the 2D idealized simulation (the figure is not shown).